Remotion/Web/Core/Res/Themes/NovaViso/HTML/DropDownMenu.css

```diff
@@ -106,18 +106,18 @@ ul.DropDownMenuOptions li a
   grid-template-columns: auto minmax(0, 1fr);
   grid-template-areas: "img span";
   gap: var(--remotion-themed-spacing);
-  padding: var(--remotion-themed-spacing);
+  padding: calc(var(--remotion-themed-spacing) - var(--remotion-themed-border-width));
```

Comment:
for the full-width paddings, subtracting the border-width is helpful to better stick to the intended layout. this should still not be a global subtraction from the spacing since this is used accross the entire css. It may be helpful to introduce a `--remotion-themed-spacing-with-border`, though.